.github/workflows/windows_vs2019_debug.yml

```diff
@@ -52,10 +53,13 @@ jobs:
     if: "!needs.smart_ci.outputs.skip_workflow"
     uses: ./.github/workflows/job_build_windows.yml
     with:
-      runner: 'aks-win-16-cores-32gb'
+      runner: 'aks-win-32-cores-128gb'
       affected-components: ${{ needs.smart_ci.outputs.affected_components }}
       build-type: 'Debug'
       target-branch: ${{ needs.smart_ci.outputs.target_branch }}
+      cmake-options: |-
+        -G "Ninja" `
+        -DENABLE_PYTHON=OFF
```

Comment:
I think we need to define all options, while currently it's being added to hardcoded ones.

Reply:
do you suggest to duplicate them?

Reply:
yes, as it's done for Linux